src/types/index.d.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable camelcase */
+import type { GridLayerOptions, Coords, DoneCallback } from "leaflet";
+
+export type PixelValueToColorFn = (values: number[]) => string;
+
+export interface GeorasterLayerOptions extends GridLayerOptions {
+  georasters: Georaster[];
+  georaster: Georaster;
+  resolution?: number;
+  debugLevel?: 0 | 1;
+  pixelValuesToColorFn?: PixelValueToColorFn;
+}
+
+export type GetRasterOptions = {
+  tileNwPoint: any;
+  heightOfSampleInScreenPixels: any;
+  widthOfSampleInScreenPixels: any;
+  coords: any;
+  numberOfSamplesAcross: any;
+  numberOfSamplesDown: any;
+  ymax: any;
+  xmin: any;
+};
+
+export interface DrawTileOptions {
+  tile: HTMLCanvasElement;
+  coords: Coords;
+  context: CanvasRenderingContext2D;
+  done: DoneCallback;
+}
+
+// note: Tile is taken from leaflets `InternalTiles` type and should not be modified.  - SFR 2021-01-19
+export type Tile = {
+  active?: boolean;
+  coords: Coords;
+  current: boolean;
+  el: HTMLCanvasElement;
+  loaded?: Date;
+  retain?: boolean;
+};
+
+export type GeorasterSource = {
```

Comment:
I think we can remove GeorasterSource because that's used by a georaster dependency (geotiff.js) and not georaster or georaster-layer-for-leaflet.